stats/opencensus/opencensus.go

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package opencensus implements opencensus instrumentation code for gRPC-Go
+// clients and servers.
+package opencensus
+
+import (
+	"context"
+	"go.opencensus.io/trace"
```

Comment:
Nit: please organize your imports; something like:

```go
import (
	"std"
	"lib"
	"imports"

	"other"
	"imports"

	renamed "imports"
)
```

Reply:
I always see a space between std lib imports and others. However, I sometimes see non grpc imports separated from grpc imports, and renamed imports never separate, only thing I've seen after gRPC imports are something like proto imports. I put spaces in between std lib, other, and grpc.

Reply:
Yeah, there are no hard-set rules everyone follows, and without a linter for it, we'll never enforce any particular rule in our repo.  It's definitely a widely used convention to separate stdlib imports from others, so as long as we do that I'm fine with whatever other preferences people have.

Reply:
Yeah that seems like the most important/always followed distinction.